packages/back-end/src/routers/demo-datasource-project/demo-datasource-project.controller.ts

```diff
@@ -148,25 +147,23 @@ const DEMO_FEATURES: Omit<
             enabled: true,
           },
           {
-            type: "experiment",
+            type: "experiment-ref",
             description: "",
             id: `${getDemoDataSourceFeatureId()}-exp-rule`,
-            trackingKey: getDemoDataSourceFeatureId(),
-            hashAttribute: "user_id",
-            coverage: 1,
             enabled: true,
-            values: [
+            experimentId: getDemoDataSourceFeatureId(), // This value is replaced below after the experiment is created.
+            variations: [
               {
-                value: "current",
-                weight: 0.3334,
+                value: "v0",
+                variationId: "current",
```

Comment:
I think these are swapped.  `variationId` should be equal to the experiment `variations[i].id` field and `value` should be the string value of the feature flag for that variation.

Reply:
Swapped them in this change f647b61c